src/utils/validateName.ts

```diff
@@ -103,16 +99,15 @@ export function validateIP(value: string): string | void {
 
 export function validateMountPoint(value: string): string | void {
   if (value === "") return "Mount point is required";
-  if (!UNIX_PATH_REGEX.test(value)) return "Invalid path";
   value = value.trim();
   if (value === "" || value === "/" || !value.startsWith("/"))
     return "Mount Point must start '/' and can't be positioned at root('/')";
 }
 
 export function validateDiskName(value: string): string | void {
   if (value === "") return "Disk Name is required";
-  if (!ALPHANUMERIC_UNDERSCORE_REGEX.test(value)) return "Invalid name";
-  if (value.length > 32) return "Name must be at most 15 characters";
+  if (!ALPHANUMERIC_UNDERSCORE_REGEX.test(value)) return "Invalid disk name";
+  if (value.length > 32) return "Name must be at most 32 characters";
```

Comment:
grid3 client has limitations on disk name to not be more than 15, this would fail